src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -39,7 +39,7 @@ def migrate_views(self, ctx: RuntimeContext):
     @job_task(job_cluster="table_migration", depends_on=[migrate_views])
     def refresh_migration_status(self, ctx: RuntimeContext):
         """Refresh the migration status to present it in the dashboard."""
-        ctx.tables_migrator.index()
+        ctx.tables_migrator.index_full_refresh()
```

Comment:
@nkvuong : Use this method in the other workflows as well

Reply:
done 👍